sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+        public static implicit operator Azure.ResourceManager.Sphere.Models.CertificateStatus (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.CertificateStatus left, Azure.ResourceManager.Sphere.Models.CertificateStatus right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ClaimDevicesContent
+    {
+        public ClaimDevicesContent(System.Collections.Generic.IEnumerable<string> deviceIdentifiers) { }
+        public System.Collections.Generic.IList<string> DeviceIdentifiers { get { throw null; } }
+    }
+    public partial class CountDeviceResponse : Azure.ResourceManager.Sphere.Models.CountElementsResponse
+    {
+        internal CountDeviceResponse() { }
+    }
+    public partial class CountElementsResponse
+    {
+        internal CountElementsResponse() { }
+        public int Value { get { throw null; } }
+    }
+    public partial class DeviceGroupPatch
+    {
+        public DeviceGroupPatch() { }
+        public Azure.ResourceManager.Sphere.Models.AllowCrashDumpCollection? AllowCrashDumpsCollection { get { throw null; } set { } }
+        public string Description { get { throw null; } set { } }
+        public Azure.ResourceManager.Sphere.Models.OSFeedType? OSFeedType { get { throw null; } set { } }
+        public Azure.ResourceManager.Sphere.Models.RegionalDataBoundary? RegionalDataBoundary { get { throw null; } set { } }
+        public Azure.ResourceManager.Sphere.Models.UpdatePolicy? UpdatePolicy { get { throw null; } set { } }
+    }
+    public partial class DeviceInsight
+    {
+        internal DeviceInsight() { }
+        public string Description { get { throw null; } }
+        public string DeviceId { get { throw null; } }
+        public System.DateTimeOffset EndTimestampUtc { get { throw null; } }
+        public string EventCategory { get { throw null; } }
+        public string EventClass { get { throw null; } }
+        public int EventCount { get { throw null; } }
+        public string EventType { get { throw null; } }
+        public System.DateTimeOffset StartTimestampUtc { get { throw null; } }
+    }
+    public partial class DevicePatch
+    {
+        public DevicePatch() { }
+        public string DeviceGroupId { get { throw null; } set { } }
+    }
+    public partial class GenerateCapabilityImageContent
+    {
+        public GenerateCapabilityImageContent(System.Collections.Generic.IEnumerable<Azure.ResourceManager.Sphere.Models.CapabilityType> capabilities) { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.Sphere.Models.CapabilityType> Capabilities { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ImageType : System.IEquatable<Azure.ResourceManager.Sphere.Models.ImageType>
```

Comment:
This name is quite generic, could we rename it to be more specific?

Reply:
fixed in latest commit